image.adoc

```diff
@@ -40,5 +40,6 @@ https://image.izettle.com/swagger
 ```
 Where `o` is a `2000*2000px` and `L` is a `560*560px` image.
 
+To apply this uploaded image on a product, use the first URL provided in the `imageUrls` array.
```

Comment:
@pp-svanstrom Would you say this is correct?

Reply:
Yes 👍